unit_tests/src/test_atomic_apply_acl_operations.cpp

```diff
@@ -432,6 +433,209 @@ TEST_CASE("#7338")
     CHECK(json_error_string == nullptr);
 }
 
+TEST_CASE("Atomic acls respect entity zone")
+{
+    using namespace std::string_literals;
+
+    rodsEnv env;
+    _getRodsEnv(env);
+    irods::experimental::client_connection conn;
+    const auto user_home = fs::path{static_cast<char*>(env.rodsHome)}.string();
```

Comment:
Based on the test, it looks like L443 can be replaced with the following:
```cpp
const auto* const user_home = env.rodsHome;
```
Or ...
```cpp
const std::string user_home = env.rodsHome;
```

Please verify and rerun this one unit test to make sure it still passes.